lib/rspec/core/metadata.rb

```diff
@@ -130,7 +130,7 @@ def populate
 
           metadata[:block]            = block
           metadata[:description_args] = description_args
-          metadata[:description]      = build_description_from(*metadata[:description_args])
+          metadata[:description]      = build_description_from(*description_args)
```

Comment:
I think this changes the behaviour, as the metadata built up here includes child description_args and not just the passed in ones.